apps/rewards/app/components/Panel/ViewReward.js

```diff
@@ -0,0 +1,258 @@
+  delay: 0
+
+}
+
+
+
+class ViewReward extends React.Component {
+    static propTypes = {
+      reward: PropTypes.object,
+    }
+    renderDescription = (description = '') => {
+      // Make '\n's real breaks
+      return description.split('\n').map((line, i) => (
+        <React.Fragment key={i}>
+          {line}
+          <br />
+        </React.Fragment>
+      ))
+    }
+    render() {
+      const {
+        creator,
+        isMerit,
+        referenceToken,
+        rewardToken,
+        amount,
+        startDate,
+        endDate,
+        description,
+        delay,
+      } = reward
+      const truncatedCreator = `${creator.slice(0, 6)}...${creator.slice(-4)}`
+      const translateToken = (token) => {
+        if (token == 0x0) {
+          return 'ETH'
+        }
+      }
+      return (
+        <div>
+          <SidePanelSplit>
+            <div>
+              <h2>
+                <Label>Created by</Label>
+              </h2>
+              <Creator>
+                <CreatorImg>
+                  <Blockies seed={creator} size={8} />
+                </CreatorImg>
+                <div>
+                  <p>
+                    {/* // TODO: Change to etherscanUrl constant for the selected network*/}
+                    <SafeLink
+                      href={`https://rinkeby.etherscan.io/address/${creator}`}
+                      target="_blank"
+                      title={creator}
+                    >
+                      {truncatedCreator}
+                    </SafeLink>
+                  </p>
+                </div>
+              </Creator>
+            </div>
+            <div>
+              <h2>
+                <Label>Status</Label>
+              </h2>
+              <div>
+                {
+                  (
+                    Date.now() > endDate
+                  ) ? (
+                      <div>
+                        <IconCheck /> Available
+                      </div>
+                    ) : (
+                      <div>
+                        <IconTime /> Pending
+                      </div>
+                    )
+                }
+              </div>
+            </div>
+          </SidePanelSplit>
+          {description && (
+            <Part>
+              <React.Fragment>
+                <h2>
+                  <Label>Description:</Label>
+                </h2>
+                <p>{this.renderDescription(description)}</p>
+              </React.Fragment>
+            </Part>
+          )}
+          <SidePanelSplit>
+            <div>
+              <h2>
+                <Label>Reference Asset</Label>
+              </h2>
+              <p>{referenceToken}</p>
+            </div>
+            <div>
+              <h2>
+                <Label>Type</Label>
+              </h2>
+              <div>
+                {isMerit?'Merit Reward':'Dividend Reward'}
+              </div>
+            </div>
+          </SidePanelSplit>
+          <SidePanelSplit>
```

Comment:
Needs to be:
`        </SidePanelSplit>
`          <SidePanelSplit style={{ borderTop: '0' }}>

or else there is double-width border.

Reply:
Thanks! I was wondering about the best way around this